eidpdata/src/main/java/uk/ac/ucl/eidp/data/PoolStrategy.java

```diff
@@ -172,8 +163,10 @@ public void setProperties(Properties p) {
   }
 
   @Override
-  public List<Map<String, String>> processDbTransaction(String methodPath, Map<String, String> parameters) {
-    throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
+  public List<Map<String, String>> processDbTransaction(
+    String methodId, 
+    Map<String, String> parameters) {
```

Comment:
\[Checkstyle\] WARNING: 'Map' have incorrect indentation level 4, expected level should be 6\.